version/version_test.go

```diff
@@ -158,7 +159,21 @@ func Test_runVersionCmd(t *testing.T) {
 
 	info := version.NewInfo()
 	stringInfo, err := json.Marshal(info)
+
+	extraInfo := &version.ExtraInfo{"key1": "value1"}
+	ctx := context.WithValue(context.Background(), version.ContextKey{}, extraInfo)
```

Comment:
Why is there a mixture of `assert` and `require` calls? It should be latter only.